Project_4

NailyaBorkova opened the pull request:
Projet_4

NailyaBorkova committed:
Project_4

mkouznetsov-git commented on src/test/java/AccrodTextTest.java:
```diff
@@ -0,0 +1,102 @@
+
+}
+
+enum Browser {
```
Этот Enum нужно вынести в отдельный файл, как отдельный класс. Один файл должен содержать один класс. В крайнем случае, могут быть вложенные классы в один родительский, но это здесь не требуется.

NailyaBorkova committed:
Project_4, правки по замечаниям.
Enum изучала по вебинару и в интернете, поэтому пока не до конца усвоила правила работы с ним. В следующем спринте он как раз есть. Спасибо.
